src/Shared/Recruit.Vacancies.Client/Domain/Entities/AlertType.cs

```diff
@@ -3,6 +3,7 @@
     public enum AlertType
     {
         TransferredVacancies,
-        BlockedProvider
+        BlockedProvider,
```

Comment:
The names could be improved to show the effect and reason for example:  VacanciesClosedAsProviderBlocked, VacanciesTransferredAsProviderBlocked

Reply:
👍 I'll review this at the end.